src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -252,6 +252,10 @@ export class EditConcept extends Component {
       if (modifyMap.url === url) {
         modifyMap[name] = value;
       }
+      if (name === 'source') {
```

Comment:
Shouldn't we have a constant for this?

Reply:
I have replaced the expression with a constant.